lib/python/Plugins/Extensions/FileCommander/plugin.py

```diff
@@ -936,8 +936,8 @@ def keyGoBottom(self):
 	def keyHashes(self, path=None):
 		if path is None:
 			path = self.sourceColumn.getPath()
-		if isfile(path):
-			if stat(path).st_size < HASH_CHECK_SIZE:
+		if isfile(path) or islink(path):
+			if lstat(path).st_size < HASH_CHECK_SIZE:
```

Comment:
This is wrong if you ask me.

Reply:
islink is true for a directory link and this is wrong.
Creating a hash from a link makes no sense.

Reply:
We can think about to create the hash from the link target if the link points to a file.